### PR TITLE
[zh] update access-cluster-api.md

### PR DESCRIPTION
**Pull request by Iceber**
en: https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/administer-cluster/access-cluster-api.md#go-client 

translated access-api-from-pod pr: https://github.com/kubernetes/website/pull/26950

### CODE REVIEW

**Comment by tengqm** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this **on the master branch**:

```shell
./scripts/lsync.sh content/zh/docs/tasks/administer-cluster/access-cluster-api.md
```

**Comment by Iceber** [comment]
@tengqm updated. Thanks

File: content/zh/docs/tasks/administer-cluster/access-cluster-api.md [hunk 1]
```diff
@@ -251,11 +251,13 @@ Kubernetes 官方支持 [Go](#go-client)、[Python](#python-client)、[Java](#ja
 #### Go 客户端  {#go-client}
 
 <!--
-* To get the library, run the following command: `go get k8s.io/client-go/<version number>/kubernetes` See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
-* Write an application atop of the client-go clients.-->
+* To get the library, run the following command: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>` See [https://github.com/kubernetes/client-go/releases](https://github.com/kubernetes/cli    ent-go/releases) to see which versions are supported.
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/access-cluster-api.md:254]: the second link has typo

**Review by tengqm — COMMENTED** [review]
/approve
looks good other than a nit.

**Comment by Iceber** [comment]
@tengqm  updated, Thanks

File: content/zh/docs/tasks/administer-cluster/access-cluster-api.md [hunk 2]
```diff
@@ -339,9 +340,9 @@ for i in ret.items:
 #### Java 客户端    {#java-client}
 
 <!--
-* To install the [Java Client](https://github.com/kubernetes-client/java), simply execute :
+To install the [Java Client](https://github.com/kubernetes-client/java), simply execute :
```
Comment (zhiguo-lu) [inline content/zh/docs/tasks/administer-cluster/access-cluster-api.md:343]:
```suggestion
To install the [Java Client](https://github.com/kubernetes-client/java), run:
```

Comment (zhiguo-lu) [reply content/zh/docs/tasks/administer-cluster/access-cluster-api.md:343]: Sync with the latest English version

**Review by zhiguo-lu — COMMENTED** [review]
挑了你一个小毛病，注释掉的英文和中文不同步。

File: content/zh/docs/tasks/administer-cluster/access-cluster-api.md (hunk 2 above)
Comment (Iceber) [reply content/zh/docs/tasks/administer-cluster/access-cluster-api.md:343]: Thanks

**Comment by zhiguo-lu** [comment]
/lgtm
thanks for your work.

**Pull request merged**